plugins/module_utils/apply.py

```diff
@@ -131,14 +138,18 @@ def k8s_apply(resource, definition, **kwargs):
     existing, desired = apply_object(resource, definition)
     server_side = kwargs.get("server_side", False)
     if server_side:
-        body = json.dumps(definition).encode()
+        versions = gather_versions()
+        body = definition
+        if LooseVersion(versions["kubernetes"]) < LooseVersion("25.0.3"):
```

Comment:
I'm assuming you meant to put 25.3.0 here? We should probably just use 25.0.0 though, as this is likely more correct.

Reply:
Yes, will do